### ISSUE TITLE
This pipeline work only with human and mouse database?

### ISSUE THREAD

**Issue by sajjadasaf** [GaryZhangYue/mir-seek#1]
This pipeline work only with human and mouse database?

**Comment by GaryZhangYue** [comment]
> This pipeline work only with human and mouse database?

Yes, I think so. We have an updated pipeline here https://github.com/OpenOmics/mir-seek. Although it also only supports human and mouse, it is a more completed version.
